group.js

```diff
@@ -12,7 +12,13 @@ class Group {
 
   name() {
     if (this.activityCode.isActivity()) {
-      return this.room.name.split(' ')[0] + ' ' + this.activityCode.groupNumber
+      // This is not ideal; when displaying room names we'd like to
```

Comment:
Good find! What is your preferred name for groups beginning with Salle / Halle / etc? I imagine that the current naming scheme isn't ideal for French Championship rooms ("Halle sportive - Zone 1 1" isn't that easy to read :) ).

Two more complicated ideas are:
- Look at all of the names of rooms, and drop any repeated words
- Add a function like IgnoreWordForGroupNames("Room"), so that you can ignore Salle and I can ignore Room.

But this should hopefully be good enough for you for now.

Reply:
I wouldn't bother too much to adapt the code to various languages just yet, as long as it doesn't hide the useful part of the stage/room I'm fine with the name :)

Also for real multi stage competitions we'll likely expect international crowd and just use "[color] Stage"!

Reply:
(marking this as resolved since no further action is needed)